librz/include/rz_util/rz_str.h

```diff
@@ -10,6 +10,8 @@
 extern "C" {
 #endif
 
+#define ASCII_LAST_CODE_POINT 0x7F
```

Comment:
be sure this is the correct place. i remember there was a header with some constants, maybe it could go there, also since this is exposed it should have `RZ_` prefix

Reply:
Yeah, I left it there because it not really is Unicode, and there is not ASCII header. But putting it to `rz_unicode.h` is indeed better.